packages/schematics/angular/workspace/files/tsconfig.json.template

```diff
@@ -7,7 +7,6 @@
     "declaration": false,
     "module": "esnext",
     "moduleResolution": "node",
-    "emitDecoratorMetadata": true,
```

Comment:
`emitDecoratorMetadata` is required for libraries. I think when removing this we should add `emitDecoratorMetadata` in the library tsconfig.

Reply:
I was hoping to add the transformer to library compilation as well but was unable to find an extension point within ng-packgr.

Added the option to the library schematic pending TS transformer support for ng-packgr.